evalml/data_checks/__init__.py

```diff
@@ -7,7 +7,7 @@
 from .data_check_message import DataCheckMessage, DataCheckWarning, DataCheckError
 from .data_check_message_type import DataCheckMessageType
 from .default_data_checks import DefaultDataChecks
-from .invalid_targets_data_check import InvalidTargetDataCheck
+from .invalid_target_data_check import InvalidTargetDataCheck
```

Comment:
I'm not sure, but would this count as a breaking change?

Reply:
Oo good call! I'd say it's safer to add it as a breaking change since we're changing the module name. Adding to release notes, thanks!